lib/internal/test_runner/test.js

```diff
@@ -124,8 +124,8 @@ class TestContext {
   }
 
   test(name, options, fn) {
-    // eslint-disable-next-line no-use-before-define
```

Comment:
Speaking only for the test runner changes - I don't care for this and would prefer to leave things as they are.